### PR TITLE
[INLONG-9976][Manager] Support multiple types of audit indicator queries

### LINKED ISSUE #9976: [Improve][Manager] Support multiple types of audit indicator queries
### Description

Support multiple types of audit indicator queries.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request

- Fixes #9976

### Motivation

Support multiple types of audit indicator queries.

### Modifications

Support multiple types of audit indicator queries.

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java [hunk 1]
```diff
@@ -133,9 +134,7 @@ public class AuditServiceImpl implements AuditService {
     private static final String TERMS = "terms";
 
     // key: type of audit base item, value: entity of audit base item
-    private final Map<String, AuditBaseEntity> auditSentItemMap = new ConcurrentHashMap<>();
-
-    private final Map<String, AuditBaseEntity> auditReceivedItemMap = new ConcurrentHashMap<>();
+    private final Map<String, Map<Integer, AuditBaseEntity>> auditIndicatorMap = new ConcurrentHashMap<>();
```
Comment (aloyszhang) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java:137]: doc should keep consistent  with code

Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java]: Done.

**Review by vernedeng — APPROVED** [review]
LGTM

**Review by aloyszhang — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**